### PR TITLE
feat: (OPI CNI) This is the initial implementation of OPI CNI

### PR DESCRIPTION
**Pull request by mardim91**
Initial implementation of OPI CNI. Currently only Intel Mt.Evans IPUs are supported by this CNI. Target is to support more IPUs/DPUs in the future.  It has an integration with the [EVPN GW API](https://github.com/opiproject/opi-api/tree/main/network/evpn-gw) .

### CODE REVIEW

**Review by nupurjai — COMMENTED** [review]
Is this based on multus and secondary interface?

**Comment by mardim91** [comment]
> Is this based on multus and secondary interface?

Yes it is

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,25 @@
 # SPDX-License-Identifier: Apache-2.0
-# Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
```
Comment (glimchb) [inline Dockerfile:2]: you should not remove copyright

**Review by glimchb — CHANGES_REQUESTED** [review]
in review

**Comment by mardim91** [comment]
@glimchb In the last tsc meeting we have decided this is not the right repo to put the cni. Also the name is a bit confusing because this cni integrates directly with evpn gw api which is a use case API. So we have decided to name it differently and use the OPI-PoC repo for now. So no need to review this PR. I will close it.

**Comment by glimchb** [comment]
> @glimchb In the last tsc meeting we have decided this is not the right repo to put the cni. Also the name is a bit confusing because this cni integrates directly with evpn gw api which is a use case API. So we have decided to name it differently and use the OPI-PoC repo for now. So no need to review this PR. I will close it.

we can change the name, but still need new repo for it

**Pull request closed without merging**